### PR TITLE
probers: log to file 

### PR DESCRIPTION
**Pull request by oxddr**
ref https://github.com/kubernetes/kubernetes/issues/87143
/assign @mm4tt

### CODE REVIEW

File: clusterloader2/pkg/measurement/common/probes/manifests/dnsLookup/dns-prober-deployment.yaml [hunk 1]
```diff
@@ -25,6 +25,8 @@ spec:
             # TODO(oxddr): according to @wojtek-t there are differences between fully and not fully qualified domain names
             # Investigate it and potentially measure latency for both
             - --dns-probe-url=ping-server.probes
+            - --logtostderr=false
+            - --log_file=/var/log/dns-lookup.log
```
Comment (mm4tt) [inline clusterloader2/pkg/measurement/common/probes/manifests/dnsLookup/dns-prober-deployment.yaml]: You need to add a volume and a volumeMount, e.g. see how it's done for hollow node: https://github.com/kubernetes/kubernetes/blob/aacd79cc36347bd56d22169de23a5156761dacd6/test/kubemark/resources/hollow-node_template.yaml#L129

**Comment by mm4tt** [comment]
/lgtm

This LGTM, but let's wait for the https://github.com/kubernetes/test-infra/pull/15895 to get first so we can verify that everything works in the presubmit

/hold

**Comment by oxddr** [comment]
PTAL

**Comment by mm4tt** [comment]
/lgtm
/approve

/hold
Wait for https://github.com/kubernetes/kubernetes/pull/88113

**Comment by mm4tt** [comment]
/test pull-perf-tests-clusterloader2
--

**Comment by oxddr** [comment]
Logs has been successfully collected in the presubmit's test (e.g.
https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/perf-tests/981/pull-perf-tests-clusterloader2/1230808617817477120/artifacts/e2e-981-62db2-minion-group-m4pp/cl2-dns-lookup.log
)

**Comment by mm4tt** [comment]
Nice!

**Pull request merged**